app/src/main/res/layout/list_playlist_grid_item.xml

```diff
@@ -67,4 +67,14 @@
         tools:ignore="RtlHardcoded"
         tools:text="Uploader" />
 
+    <CheckBox
+        android:id="@+id/selectCheckBox"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="-163dp"
+        android:layout_toEndOf="@+id/itemUploaderView"
+        android:buttonTint="@color/black_settings_accent_color"
+        android:clickable="false"
+        android:visibility="invisible" />
```

Comment:
Why invisible? It should be `gone` when you are not in multiselection mode, otherwise the checkbox will use up space even when it's not needed
(also in other places)